terminal/terminal.go

```diff
@@ -33,6 +33,25 @@ const (
 	MouseExtSGR
 )
 
+type WindowManipulationInterface interface {
+	RestoreWindow(term *Terminal) error
+	IconifyWindow(term *Terminal) error
+	MoveWindow(term *Terminal, pixelX int, pixelY int) error
+	ResizeWindowByPixels(term *Terminal, pixelsHeight int, pixelsWidth int) error
+	BringWindowToFront(term *Terminal) error
+	//SendWindowToBack(term *Terminal) error
```

Comment:
Should commented function names be removed?

Reply:
@MaxRis Will remove them